lib/logstash/codecs/cef.rb

```diff
@@ -348,51 +348,25 @@ def encode(event)
   # Escape pipes and backslashes in the header. Equal signs are ok.
   # Newlines are forbidden.
   def sanitize_header_field(value)
-    output = String.new
-
-    value = value.to_s.gsub(/\r\n/, "\n")
-
-    value.each_char{|c|
-      case c
-      when "\\", "|"
-        output << "\\#{c}"
-      when "\n", "\r"
-        output << " "
-      else
-        output << c
-      end
-    }
-
-    return output
+    value.to_s
+         .gsub(/\r\n/, "\n")
```

Comment:
```suggestion
         .gsub("\r\n", "\n")
```
... does a String#index and avoids regexp operation (since JRuby 9.2.9)

Reply:
same bellow .. GH UI has messed (lost) my second suggestion :unamused:

Reply:
done.